lesson-1-intro-to-k8s/README.md

```diff
@@ -199,9 +200,9 @@ $ curl http://0.0.0.0:9999
 
 ## Cluster access
 
-Link: https://console-openshift-console.apps.cluster-4svlt.4svlt.sandbox1027.opentlc.com
+If you need a cluster, you can use the Developer Sandbox mentioned above.
```

Comment:
... "For basic experiments with vanilla Kubernetes, have a look at https://kind.sigs.k8s.io/"